cpu/sensors.py

```diff
@@ -33,6 +33,8 @@ class Sensors(Test):
 
     """
     Run sensors command on ppc architectures.
+
+    :avocado: tags=cpu,power,privileged
```

Comment:
Right, I missed the conditional on the distro.  So, let's fix the docstring and the tag.  Thanks!